### PR TITLE
Upsert now stop double counting of test scores

### PR DESCRIPTION
**Pull request by wavell**
# Upsert now stop double counting of test scores

## Description
Upsert now stop double counting of test scores

## Issues:
Refs: #159

## How has this been tested:
 - [ ] Covered by existing integration testing
 - [x] Added integration testing to cover
 - [ ] Verified all A/C passes
     * [ ] develop
     * [ ] master
     * [ ] tag/other branch
 - [ ] Test environment
    * [ ] Shared Packet K8s cluster
    * [ ] New Packet K8s cluster
    * [ ] Kind cluster
 - [ ] Have not tested

## Types of changes:
 - [x] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
 - [ ] Documentation update

## Checklist:
**Documentation**
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] No updates required.

**Code Review**
- [ ] Does the test handle fatal exceptions, ie. rescue block

**Issue**
- [ ] Tasks in issue are checked off

### CODE REVIEW

File: spec/utils/utils_spec.cr [hunk 1]
```diff
@@ -82,14 +82,26 @@ describe "Utils" do
...
+      YAML.parse(file)
+    end
+    # puts yaml["items"].as_a.inspect
+    (yaml["items"].as_a.find {|x| x["name"] == "liveness" && x["points"] == task_points("liveness")}).should be_truthy
+  end
+
   it "'upsert_task' should find and update an existing task in the file"  do
     create_results_yml
     upsert_task("liveness", PASSED, task_points("liveness"))
+    upsert_task("liveness", PASSED, task_points("liveness"))
```
Comment (williscool) [inline spec/utils/utils_spec.cr:98]: @wavell is this supposed to be here twice?

File: src/tasks/utils/utils.cr [hunk 2]
```diff
@@ -255,8 +255,12 @@ def upsert_task(task, status, points)
   results = File.open("#{LOGFILE}") do |f| 
     YAML.parse(f)
   end 
-  found = false
+  # found = false
```
Comment (williscool) [inline src/tasks/utils/utils.cr:258]: why not just delete this line?

Comment (wavell) [reply src/tasks/utils/utils.cr]: Deleted

File: spec/utils/utils_spec.cr (hunk 1 above)
Comment (wavell) [reply spec/utils/utils_spec.cr:98]: Yes,  it tests if there is a double entry if the points are the same (upsert)

**Pull request merged**